### PR TITLE
task 11 done

### PR DESCRIPTION
**Pull request by SamenaBasith**

### CODE REVIEW

File: models/model.js [hunk 1]
```diff
@@ -65,8 +65,18 @@ AS comment_count
 exports.selectArticlesById = (article_id) => {
 
 let querySQL = 
-`SELECT * FROM articles 
-WHERE article_id = $1;`
+// `SELECT * FROM articles
```
Comment (mallionaire) [inline models/model.js]: Nope! 👎 
Please don't commit commented out code 😄

File: __tests__/app.test.js [hunk 2]
```diff
@@ -424,3 +424,28 @@ describe("GET /api/users", () => {
...
+                topic: "mitch",
+                created_at: "2020-07-09T20:11:00.000Z",
+                votes: 100,
+                comment_count: "11"
+            })
+          );
+        });
+      });
+  });
+
```
Comment (mallionaire) [inline __tests__/app.test.js:471]: I'd _quite_ like to see a test for requesting an article with no associated comments, just to be doubly sure any error handling is working correctly... 🤔

**Review by mallionaire — APPROVED** [review]
Have a careful think about adding an additional test for an article with no comments, more details below. 

But really nice job!

**Review by leekli — APPROVED** [review]
Great work Samena, thanks for implementing the requested changes.

Well done you are free to merge this branch.

**Pull request merged**